fat-jar.html

```diff
@@ -0,0 +1,198 @@
+<h2>Creating a Fat Jar</h2>
+
+    
+<p>
+    You can create a runnable fat jar for your JavaFX 11 project over the
```

Comment:
In general, I think we want to say that this is the wrong approach, and that people should start using modules (in their own application and in their libraries) so that they can use jlink.
We need a big disclaimer on the start of this page, stating this. I am pretty sure Mark Reinhold and Kevin will freak out otherwise :)

Reply:
> In general, I think we want to say that this is the wrong approach

Indeed. :)

Reply:
I fully agree, but people are still demanding/used to fat jars, mainly because they haven't moved to modular yet (and jlink won't work). I'll clarify.

Reply:
I've modified the document for the fat jar.
I've added two main warnings, and moved command line to the end, after Maven and Gradle.